src/Microsoft.ServiceFabric.Services.Remoting/Client/ServiceProxyFactory.cs

```diff
@@ -91,65 +91,45 @@ public ServiceProxyFactory(OperationRetrySettings retrySettings = null)
         {
             var serviceInterfaceType = typeof(TServiceInterface);
 
-#if !DotNetCoreClr
-
             //Use provider to find the stack
-            if (this.proxyFactoryV1 == null && this.proxyFactoryV2 == null)
+            var provider = this.GetProviderAttribute(serviceInterfaceType);
+
+#if !DotNetCoreClr
```

Comment:
I assume V1 listener is not available on netcore. Is this correct?

Reply:
yes, this is correct

Reply:
Thanks (also thank for all your replies :smile:)